inc/utils/namespace.php

```diff
@@ -1145,10 +812,19 @@ function clickhouse_query( string $query, string $body = '', string $return = 'a
 		$request_args['body'] = $body;
 	}
 
+	// Add query parameters.
+	if ( ! empty( $params ) ) {
+		$param_keys = array_map( function ( $key ) {
+			return "param_{$key}";
+		}, array_keys( $params ) );
+		$params = array_combine( $param_keys, array_values( $params ) );
+		$clickhouse_url = add_query_arg( urlencode_deep( $params ), $clickhouse_url );
```

Comment:
I think this could be clearer with a foreach:

```suggestion
		$prepared = [];
		foreach ( $params as $key => $value ) {
			$prepared[ 'param_' . urlencode( $key ) ] = urlencode_deep( $value );
		}
		$clickhouse_url = add_query_arg( $prepared, $clickhouse_url );
```

`$params` should maybe also be json encoded or something?

Reply:
Why json encoded? I'm not sure what you mean

Reply:
eg for a query like:
```php
Clickhouse\query(
    'SELECT * FROM analytics WHERE blog_id IN {blogs:Array(UInt16)}',
    [
         'blogs' => [ 1, 2, 3, 4 ],
    ]
)
```

Right now, you'll need to turn the array into `[1,2,3,4]` in your calling function; should `query()` handle this itself?

Reply:
(I'm not sure the format is actually JSON btw, so the answer is probably to make it the caller's problem)

Reply:
Can we keep it simple for now? There are bigger fish to fry :)

Reply:
Figured it out. It basically wants JSON encoding but with single quotes for strings e.g. `?param_foo=['a','b']`